german/wegschneiden/ausziehen/ausziehen.md

```diff
@@ -0,0 +1,10 @@
+Sie stehen nun nackt vor ihren durchn�ssten Sachen und ihren gefunden Utensilien.
+Pl�tzlich wissen Sie was zu tun ist!
+Sie nehmen die Kleider und versuchen damit das Bootsloch mit dem Hammer und den N�geln zu flicken....
+Doch da Sie WebDeveloper sind, gl�nzen Sie nur so vor Unf�higkeit im Umgang mit allem was keine Tastatur ist und Sie zerst�ren das komplette Boot.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.